### PR TITLE
Add TransSee

### PR DESCRIPTION
**Pull request by doconnoronca**
Enable support for displaying TransSee transit predictions on TidByt with support for over 150 transit agencies. See https://www.transsee.ca/tidbyt for usage information.

Paid premium is required but a two week free trail is available for TidByt owners.

### CODE REVIEW

**Comment by doconnoronca** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by doconnoronca** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by drudge** [comment]
recheck

**Comment by doconnoronca** [comment]
recheck

**Comment by doconnoronca** [comment]
Any help on getting the CLA done?

**Comment by betterengineering** [comment]
recheck

**Comment by rohansingh** [comment]
@doconnoronca I think the problem is a mismatch between the author on the Git commits and your GitHub account. Here is the author line from the commit:
```
Author: Darwin <doconnor@transsee.ca>
```

Could you please ensure this email is associated with your GitHub account here:
https://github.com/settings/emails

Separately you'll need to run `make format` to fix the formatting of the Starlark file so it can pass the tests.

**Comment by doconnoronca** [comment]
recheck

**Comment by doconnoronca** [comment]
recheck

**Comment by doconnoronca** [comment]
I have tried to address the issues but a maintainer need to run the workflows. I don't see a way to change the label.

**Comment by rohansingh** [comment]
Sorry, I approved the tests to run now.

I see two issues:

- [ ] Our style guide requires app descriptions to end with punctuation.
- [ ] There needs to be some sort of useful preview for the app even if the user hasn't configured it yet. This is what shows up when you're browsing through the list of apps.

File: apps/transsee/transsee.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+var source []byte
+
+// New creates a new instance of the TransSee applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "transsee",
+		Name:        "TransSee",
+		Author:      "doconno@gmail.com",
+		Summary:     "Realtime transit prediction",
+		Desc:        "Provides real-time transit predictions based on actual travel times. Requires paid premium. See transsee.ca/tidbyt for usage information",
```
Comment (rohansingh) [inline apps/transsee/transsee.go:20]:
```suggestion
		Desc:        "Provides real-time transit predictions based on actual travel times. Requires paid premium. See transsee.ca/tidbyt for usage information.",
```
Add punctuation.

File: apps/transsee/transsee.star [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                ],
+            ))
+            if config.bool("scroll"):
+                col.append(render.Marquee(width = 64, child = render.Text(json["dest"])))
+            else:
+                col.append(render.Text(json["dest"]))
+
+        return render.Root(render.Column(children = col))
+    else:
+        return []
```
Comment (rohansingh) [inline apps/transsee/transsee.star:30]: This needs to return some sort of useful preview.

**Comment by doconnoronca** [comment]
Addressed the issues. Looks like the Build and Test is failing because of a problem with another app.

**Comment by doconnoronca** [comment]
recheck

**Pull request merged**